### PR TITLE
Create ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-1756](https://cumulocity.atlassian.net/browse/DM-1756)
Original PR: [7216](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7216)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: OPCUA Device protocol Radio buttons go unselect when we expand another variable (#7216)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md]: The Claude AI suggested to use the following title:
```suggestion
title: OPC-UA device protocol: Radio button selection lost when expanding another variable
```

File: content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-1756
+version: 1020.31.2
+---
+OPCUA Device protocol Radio buttons go unselect when we expand another variable (#7216)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md]: The Claude AI suggested to use the following body:
```suggestion
In the OPC-UA device protocol configuration, users could select a radio button for a specific variable. However, when expanding another variable in the same view, the previously selected radio button would become unselected. This issue has now been resolved. With this fix, the radio button selection is properly maintained even when expanding other variables in the view. Users no longer need to reselect the desired radio button after inspecting other variables.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by JacekGonera — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-1756
+version: 1020.31.2
+---
+In the OPC-UA device protocol configuration, users could select a radio button for a specific variable. However, when expanding another variable in the same view, the previously selected radio button would become unselected. This issue has now been resolved. With this fix, the radio button selection is properly maintained even when expanding other variables in the view. Users no longer need to reselect the desired radio button after inspecting other variables.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md]:
```suggestion
In the OPC UA device protocol configuration, users could select a radio button for a specific variable. However, when expanding another variable in the same view, the previously selected radio button would become unselected. This issue has now been resolved. With this fix, the radio button selection is properly maintained even when expanding other variables in the view. Users no longer need to reselect the desired radio button after inspecting other variables.
```

File: content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: OPC-UA device protocol: Radio button selection lost when expanding another variable
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md]:
```suggestion
title: Radio button selection in OPC UA device protocol configuration maintained when expanding another variable
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Oezge-Akin Please approve

**Review by Oezge-Akin — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-1756
+version: 1020.31.2
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/ui-c8y-1020-31-2-opcua-device-protocol-radio-buttons-go-unselect-when-we-expand-another.md]:
```suggestion
version: 10.20.31.2
```

**Pull request merged**